polytope/polytope.py

```diff
@@ -570,6 +564,18 @@ def _rotate(polyreg, i=None, j=None, u=None, v=None, theta=None, R=None):
     return R
 
 
+def _Givens_rotation_matrix(i, j, theta, N):
```

Comment:
Minor note about capitalization and function names (I do not find the name `givens_rotation_matrix` very readable, but for the sake of uniform naming).